packages/react-integration/cypress/integration/applicationlauncher.spec.ts

```diff
@@ -1,34 +1,6 @@
-describe('Application Launcher Demo Test', () => {
-  it('Navigate to demo section', () => {
-    cy.visit('http://localhost:3000/');
-    cy.get('#application-launcher-demo-nav-item-link').click();
-    cy.url().should('eq', 'http://localhost:3000/application-launcher-demo-nav-link');
-  });
-
-  it('Verify application launcher button', () => {
```

Comment:
This one looks like it is verifying the toggle interaction.

Reply:
Readded.